scripts/Muon/GUI/Common/fitting_widgets/basic_fitting/basic_fitting_model.py

```diff
@@ -433,6 +478,11 @@ def _reset_end_xs(self) -> None:
         """Resets the end Xs stored by the context."""
         self.fitting_context.end_xs = [self.current_end_x] * self.fitting_context.number_of_datasets
 
+    def _reset_exclude_start_and_end_xs(self) -> None:
+        """Resets the exclude start and end Xs stored by the context."""
+        self.fitting_context.exclude_start_xs = [self.current_end_x] * self.fitting_context.number_of_datasets
```

Comment:
should this be current_start_x?

Reply:
It should be `current_end_x`. I'll add a comment to make it clearer